app/views/analytics/index.html.erb

```diff
@@ -147,26 +147,59 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
-      <div class="chart-wrapper">
-        <div class="chart-title background-orange">
-          Recent Comments
-        </div>
-        <ul class="scroller">
-          <% @most_recent_comments.each do |comment| %>
-            <li>
-              <blockquote class="blockquote">
-                <p><%= link_to_case_title(comment.commentable, 25) %> -- <%= comment_created_at(comment) %></p>
-                <p><%= link_to_comment(comment, 100) %></p>
-                <small><%= link_to_comment_user(comment) %></small>
-              </blockquote>
-            </li>
-          <% end %>
-        </ul>
+    <!-- Editors List-->
+      <div class="col-sm-4">
+        <div class="chart-wrapper">
+          <div class="chart-title background-orange">
+            Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+          </div>
+          <div class="chart-stage">
+           <table class='table leader-list'>
+             <tr>
+              <th>Editor</th>
+              <th>Updates</th>
+             </tr>
+
+             <% case_updaters_count(10).each do |who| %>
```

Comment:
Don't iterate through the array - that has a performance hit.  Find a way to render this as a collection via a partial.

Reply:
Ok. I will try to find a tutorial and implement this.

Reply:
@campgurus look through our codebase for `render partial`. There should be examples of what @rlgreen91 has discussed there.